en/dev_setup/building_px4.md

```diff
@@ -256,7 +256,7 @@ make [VENDOR_][MODEL][_VARIANT] [VIEWER_MODEL_DEBUGGER_WORLD]
 - **VENDOR:** The manufacturer of the board: `px4`, `aerotenna`, `airmind`, `atlflight`, `auav`, `beaglebone`, `intel`, `nxp`, etc.
     The vendor name for Pixhawk series boards is `px4`.
 - **MODEL:** The *board model* "model": `sitl`, `fmu-v2`, `fmu-v3`, `fmu-v4`, `fmu-v5`, `navio2`, etc.
-- **VARIANT:** Indicates particular configurations: e.g. `rtps`, `lpe`, which contain components that are not present in the `default` configuration. Most commonly this is `default`, and may be omitted.
+- **VARIANT:** Indicates particular configurations: e.g. `rtps`, `lpe`, which contain components that are not present in the `default` configuration. Most commonly this is `default`, and may be omitted. 
```

Comment:
Note `rtps` config. Is there a new config? What are the options after the change?

Reply:
All `rtps` config have been removed, the `microdds_client` is included by default in most of the boards but the user has to manually start it acting on the appropriate parameters.
In simulation, the module is automatically started using udp protocol and port 8888

Reply:
@beniaminopozzan Thanks. I presume by appropriate parameters etc you mean the command line here: http://docs.px4.io/main/en/modules/modules_system.html#examples-3 ?

@Jaeyoung-Lim I guess this is the same question as https://github.com/PX4/PX4-user_guide/pull/2094/files#r1008915158 - how do we get it working on real hardware? Do you know of any example/tutorial for real hardware yet?

Wouldn't it be better if all these options were presented as  parameters so that you could configure these in the airframe file? 

If not, is the suggestion that you do so in an SD card startup file?

Reply:
@hamishwillee precisely. But on real hardware there is also the `serial_config` parameter like the old micrortps. However, I never tried it yet and it does not allow for specifying custom namespaces.